batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/controller/CustomHttpServlet.java

```diff
@@ -0,0 +1,63 @@
+package com.lftechnology.batch7crud.controller;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@SuppressWarnings("serial")
+public abstract class CustomHttpServlet extends HttpServlet {
+    private static final String ERROR_PAGE = "/WEB-INF/views/error.jsp";
+    private static final String MESSAGE = "message";
+    private static final Logger LOGGER = Logger.getLogger("CustomHttpServletLog");
+
+    protected void show404(HttpServletRequest request, HttpServletResponse response) {
+        try {
+            response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+            request.setAttribute(MESSAGE, "Page Not Found");
+            System.out.println("not found");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)